splash/kernel/inspections.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, folder=None):
         for name in files:
             full_name = os.path.join(folder, name)
             with open(full_name, "rb") as f:
-                info = json.load(f, encoding='utf8')
+                info = json.loads(f.read().decode('utf-8'), encoding='utf8')
```

Comment:
Umm.. sorry, I don't understand the issue. `decode` is converting the returned bytes to unicode. Why is that incorrect here?